example/mysql/index.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express');
+const http = require('http');
+const mysql = require('mysql');
+const terminus = require('../../lib/terminus');
+const app = express();
+const db = mysql.createConnection({
+  database: 'terminus',
+  user: 'test',
+  password: 'test'
+});
+
+app.get('/', (req, res) => {
+  db.query('SELECT 1 + 1 AS solution', (err, results) => {
+    if (err) throw err;
```

Comment:
can we either `return res.sendStatus(500)`, or `next(err)` here?

Reply:
yes, I will change this